ruby.md

```diff
@@ -4,50 +4,60 @@ Try your best to answer each question on your own before looking up the answer o
 
 
 #### 1. What is a method in Ruby? How is it different or similar to functions in JavaScript?
+Ruby methods are essentially built in functions that are used on specific datatypes. for example if you have an array of strings that you want to sort alphabetically you can just call array.sort and this built in method will alphabetically sort your array, as opposed to building a function to do the same thing from scratch.
```

Comment:
Good answer! Keep in mind though that a 'method' in Ruby is like any function in Javascript and is not restricted for use with data (an object). How does a Ruby method then differ from a Javascript function?